src/gtk/widgets/recent_info.rs

```diff
@@ -25,12 +25,13 @@ struct_Widget!(RecentInfo);
 
 impl RecentInfo {
     pub fn _ref(&self) -> Option<RecentInfo> {
-        let tmp_pointer = unsafe { ffi::gtk_recent_info_ref(GTK_RECENT_INFO(self.unwrap_widget())) };
+        let tmp_pointer = unsafe {
+            ffi::gtk_recent_info_ref(GTK_RECENT_INFO(self.unwrap_widget()))
+        };
 
-        if tmp_pointer.is_null() {
-            None
-        } else {
-            Some(gtk::FFIWidget::wrap_widget(tmp_pointer as *mut ffi::C_GtkWidget))
+        match tmp_pointer.is_null() {
```

Comment:
It kind of depends when you want to use something like: `_ => ...`. Especially when adding new entries to enum types, it would be wise to be explicit about it. However, for booleans, I do agree that cases of `false => ...` are pointless.

Coming both from a functional and an imperative background, I think that `match` is much stronger in what you can do with it, as opposed to `if`, but I don't think you can eliminate `if`-statements from the Rust language entirely. The reason why is because you often need an additional `if`-clausule for some `match`-cases.